test/Test/EventListenerIntrospectionTraitTest.php

```diff
@@ -193,6 +194,13 @@ public function testAssertListenerAtPriorityFailsWhenListenerIsNotFound()
                     $arguments['event'],
                     $arguments['priority']
                 ), $e->getMessage(), sprintf('Assertion failure message was unexpected: %s', $e->getMessage()));
+            // for PHPUnit 5.7
```

Comment:
thats the fallback for PHPUnit5, thats not just phpunit5.7

that exception class doesnt exist in phpunit 6, so there must be a fallback, other way would be a `class_alias` or a check if the class exists.